src/components/ngRedux.js

```diff
@@ -1,10 +1,41 @@
 import Connector from './connector';
+import invariant from 'invariant';
+import isFunction from '../utils/isFunction';
+import {createStore, applyMiddleware} from 'redux';
 
 export default function ngReduxProvider() {
-  let reduxStore = undefined;
-  this.setReduxStore = store => reduxStore = store;
+  let _reducer = undefined;
+  let _middlewares = [];
+  let _storeEnhancer = undefined;
 
-  this.$get = () => {
-    return Connector(reduxStore);
+  this.createStoreWith = (reducer, middlewares, storeEnhancer) => {
+  	  invariant(
+        isFunction(reducer),
+        'The reducer parameter passed to createStoreWith must be a Function. Instead received %s.',
+        typeof selector
```

Comment:
should this be:

```javascript
	  invariant(
        isFunction(reducer),
        'The reducer parameter passed to createStoreWith must be a Function. Instead received %s.',
        typeof reducer
      );
```

noticed a similar error just below here, and in connector also. Ends up having the error be `received undefined`  even if passing in object/string/etc.